mono/mini/il2cpp-stubs.cpp

```diff
@@ -650,8 +649,7 @@ Il2CppMonoClass* il2cpp_mono_class_load_from_name(Il2CppMonoImage* image, const
 
 Il2CppMonoClass* il2cpp_mono_class_try_load_from_name(Il2CppMonoImage* image, const char* name_space, const char* name)
```

Comment:
Likewise, we might be able to replace this stub with a call to `il2cpp_class_from_name`.